CONTRIBUTING.md

```diff
@@ -0,0 +1,89 @@
+# contributing
+
+This document describes how to contribute to `saturncloud/examples`.
+
+## `examples/` Directory Structure
+
+Examples in this the `examples/` directory follow a specific directory structure that allows Saturn to automatically seed them in a new user's environment with these.
+
+Each directory below `examples/` corresponds to one new Saturn Project + Jupyter that will be created in users' environments. The names of these directories should have names which only contain lowercase alphanumeric characters and dashes.
+
+```text
+.
+├── examples
+│   ├── taxi-cpu
+│   │   ├── dashboard
+│   │   │   └── dashboard.ipynb
+│   │   ├── etl
+│   │   │   └── data-loading.ipynb
+│   │   ├── README.md
+│   │   └── .saturn
+│   │       └── saturn.json
+│   │       └── start
+```
+
+### The `README.md`
+
+Each folder exactly one level below `examples/` should have a `README.md`. This should contain relevant information for understanding the example, such as:
+
+* description of any manual steps needed to use the code (like configuring credentials)
+* links to reference material like blogs posts or data dictionaries
+
+So, for example, in the directory structure above, `taxi-cpu/README.md` is required but `taxi-cpu/dashboard/README.md` is not.
+
+### The `.saturn` Directory
+
+Each example must contain a special directory called `.saturn`. This contains the details needed for Saturn to create the necessary resources for the example. This can contain the following files, and any other files will be ignored.
+
+* [`.saturn/saturn.json`](#saturn-json)
+* [`.saturn/start`](#start-script)
+
+#### `.saturn/saturn.json` <a name="saturn-json"></a>
+
+A JSON with the following structure:
+
+```json
+{
+    "image": "saturncloud/saturn:2020.07.08.1",
```

Comment:
The project name is just the name of the directory right?

Reply:
sure, makes sense

Reply:
this was agreed in #6